models/anderson_2007.cpp

```diff
@@ -2,18 +2,27 @@
 // from anderson et al 2007 - Vancomycin pharmacokinetics in preterm neonates and 
 // the prediction of adult clearance.
 [PARAM] @annotated
-CL    : 3.83 : Clearance (L/hr)
-V     : 39.4 : Volume (L) 
-WT    : 2.9  : Weight (kg)
-PMA   : 34.8 : post-menstral age (wk)
-SCR   : 0.9  : serum creatinine (mg/dL)
-VENT  : 0    : Ventillation status
-IONT  : 0    : ionotrope status
-Kage  : 0.00789 : scaling constant of age on renal function
-Fvent : 1.03 : Scaling factor applied for use of positive pressure artificial ventilation
-Fiont : 1.19 : Scaling factor applied for use of inotropes (use of dopamine)
-OCC   : 1     : Occasion
-  
+CL        : 3.83    : Clearance (L/h)
+V         : 39.4    : Volume (L) 
+WT        : 2.9     : Weight (kg)
```

Comment:
good tips. I am well aware of some of your philosophies for standardizing covariate sets - and very much agree, drives me crazy when people standarize by like the median of the value of the current trial. 

A healthy, full term baby sounds like a reasonable 'group' to target. Thanks for the advice.

The general plan is to make these models readily available for anyone else doing work with vancomycin, as well as to do a number of comparisons both across models, and between various nomograms that continue to be published. 

Secondarily, I hope this can be an example repository to demonstrate an avenue for transparency and broad access, as we aren't "shackled" by proprietary data that can't be shared, to me there should be no reason to do the entire analysis completely in the open